plugins/common/common.cpp

```diff
@@ -368,14 +368,20 @@ class StaticTextComponent : public ComponentBase
 
 	wxObject* Create(IObject *obj, wxObject *parent)
 	{
-		wxStaticText* st = new wxStaticText((wxWindow *)parent,-1,
-			obj->GetPropertyAsString(_("label")),
-			obj->GetPropertyAsPoint(_("pos")),
-			obj->GetPropertyAsSize(_("size")),
-			obj->GetPropertyAsInteger(_("style")) | obj->GetPropertyAsInteger(_("window_style")));
+		wxString label = obj->GetPropertyAsString( _("label") );
+		wxStaticText* st = new wxStaticText( (wxWindow *)parent, -1,
+			label,
+			obj->GetPropertyAsPoint( _("pos") ),
+			obj->GetPropertyAsSize( _("size") ),
+			obj->GetPropertyAsInteger( _("style") ) | obj->GetPropertyAsInteger( _("window_style") ) );
```

Comment:
Please don't include the added spaces as it destroys the diff history. Sorry for the trouble and that I didn't mention this earlier :/